client/src/components/Followers/index.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { Modal, ModalHeader, ModalBody, Button } from 'reactstrap';
+import userphoto from '../../assets/user1.png';
+import './style.scss';
+interface IState {
+    modal?: boolean;
+}
+export class Followers extends React.Component<IState> {
+    public state: IState = {
+        modal: false,
+    };
+    public toggle = (): void => {
+        this.setState({ modal: !this.state.modal });
+    }
+    public render(): JSX.Element {
+        return (
+            <div>
+                <Button color = 'danger' onClick={this.toggle}>Followers</Button>
+                <Modal className='modal-dialog-centered' isOpen={this.state.modal}>
+                    <ModalHeader toggle={this.toggle} cssModule={{'modal-title': 'w-100 text-center '}}>
```

Comment:
Why do you use cssModule?

Reply:
Cause otherwise it does not make the text in the center